common/survey/_survey/surveyDefaults.js

```diff
@@ -1,7 +1,7 @@
 const { groupNames, permissions } = require('../../auth/authGroups')
 const { keys: authGroupKeys } = require('../../auth/authGroups')
 
-const getDefaultAuthGroups = lang => [
+const defaultAuthGroups = [
```

Comment:
can it be a function that generates a new object every time? (like you did for Validator.newValidationValid?)

Reply:
it can, but it's used only when inserting a survey. basically only to insert those groups into the db. do you think the function is needed?

Reply:
I have restored the getDefaultAuthGroups method